features/SPS004_No_combination_of_containment_and_positioning.feature

```diff
@@ -0,0 +1,18 @@
+@implementer-agreement
+@SPS
+Feature: SPS004 - No combination of containment and positioning
+Each IfcProduct shall be either be contained in one and only one IfcSpatialStructureElement, or contained in one and only one IfcLinearPositioningElement/
```

Comment:
```suggestion
Each IfcProduct can either be contained in one and only one IfcSpatialStructureElement, or contained in one and only one IfcLinearPositioningElement, but not both.
```

The way I interpret the implementation below, it is only mutually exclusive for both to be present; but they both might be empty as well.

Reply:
Good point, also created a test case for that (which indeed passes without code changes).